db/db_kv_checksum_test.cc

```diff
@@ -55,87 +111,71 @@ class DbKvChecksumTest
       case WriteBatchOpType::kMerge:
         s = wb.Merge(cf_handle, "key", "val");
         break;
-      case WriteBatchOpType::kBlobIndex: {
-        // TODO(ajkr): use public API once available.
```

Comment:
My understanding is it will never be made public because the non-deprecated BlobDB does not use kBlobIndex. So this code is removed.